addons/@fumohouse/fumo/character_editor/part_preview_button.gd

```diff
@@ -0,0 +1,42 @@
+class_name PartPreviewButton
+extends Button
+
+@export var part: PartData
+
+@onready var character_viewport: CharacterViewport = %CharacterViewport
+@onready var indicator: Panel = %Indicator
+
+@onready var _fumo_appearances: FumoAppearances = FumoAppearances.get_singleton()
+
+
+func _ready():
+	var appearance := Appearance.new()
+
+	# FIXME: huh, set null defaults on appearance.config definition instead?
+	appearance.config[&"eyebrows"] = null
+	appearance.config[&"eyes"] = null
+	appearance.config[&"mouth"] = null
+
+	character_viewport.character.appearance_manager.appearance = appearance
+
+	# FIXME: Assertion failed: color is a required key in the part configuration.
+	# ...even though the same function picks a default, asset bug though?
+	if part.id == "socks_1":
+		return
```

Comment:
Please go ahead and fix `socks_1.tres` to have a default config that sets `color` to white